python/nano/scripts/bigdl-nano-init

```diff
@@ -307,6 +307,7 @@ if [ "${OPENMP}" -eq 1 ]; then
             export MKL_NUM_THREADS=$core_
             export OMP_PROC_BIND="close"
             export OMP_SCHEDULE="STATIC"
+            export MKL_DYNAMIC="FALSE"
```

Comment:
may also unset it in `bigdl-nano-unset-env`

Reply:
Will fix, thx